src/docfx/build/legacy/LegacyUtility.cs

```diff
@@ -50,7 +50,7 @@ public static string ToLegacySiteUrlRelativeToBaseSitePath(this Document doc, Do
         }
 
         public static string ToLegacyOutputPath(this LegacyManifestOutputItem legacyManifestOutputItem, Docset docset, string groupId)
-            => Path.Combine($"{groupId}", docset.Config.DocumentId.SiteBasePath, legacyManifestOutputItem.RelativePath);
+            => Path.Combine(docset.Config.DocumentId.SiteBasePath, $"{groupId}", legacyManifestOutputItem.RelativePath);
```

Comment:
>$"{groupId}" [](start = 67, length = 12)

directly `groupId`?

Reply:
groupId maybe null

---
In reply to: [240446206](https://github.com/dotnet/docfx/pull/3750#discussion_r240446206) [](ancestors = 240446206)